src/modules/company/dto/CompanyUpdate.dto.ts

```diff
@@ -57,6 +57,16 @@ export class CompanyUpdateDTO {
     @IsString()
     description?: string | null
 
+    @ApiProperty({
+        example: 'corporateReasonName',
+        type: String,
+        required: false,
+        nullable: true,
+    })
+    @IsOptional()
+    @IsString()
+    corporateReason?: string
```

Comment:
```suggestion
    companyName?: string
```
Razão social não faz sentido em inglês